card/mod/standard/set/all/rich_html/title.rb

```diff
@@ -1,13 +1,27 @@
+format do
+  view :title, closed: true, perms: :none do
+    standard_title
+  end
+
+  def standard_title
+    name_variant title_in_context(voo.title)
+  end
+end
+
 format :html do
-  view :title do
-    title = wrapped_title super()
-    title = link_to_card card.name, title if show_view? :title_link, :hide
+  # NOCACHE because alters @context_names
+  view :title, cache: :never do
```

Comment:
updated this to adhere to our new caching best practices.

Reply:
it would probably be smarter to make the name context handling part of the view api eventually.